forge-gui/res/cardsfolder/upcoming/abaddon_the_despoiler.txt

```diff
@@ -0,0 +1,8 @@
+Name:Abaddon the Despoiler
+ManaCost:2 U B R
+Types:Legendary Creature Astartes Warrior
+PT:5/5
+K:Trample
+S:Mode$ Continuous | EffectZone$ Battlefield | Condition$ PlayerTurn | Affected$ Card.YouCtrl+YouOwn+wasCastFromHand+cmcLEX | AffectedZone$ Stack | AddKeyword$ Cascade | Description$ During your turn, spells you cast from your hand with mana value X or less have cascade, where X is the total amount of life your opponents have lost this turn.
```

Comment:
add `Mark of Chaos Ascendant — ` to Description$

Reply:
also `wasCastFromYourHand` not `wasCastFromHand` – it matters in some rare situations!

Reply:
I think the YouOwn is stronger in that case, else it would have to be `wasCastFromYourHandByYou`